app/controllers/api/v1/organizations_controller.rb

```diff
@@ -0,0 +1,45 @@
+class Api::V1::OrganizationsController < Api::ApiController
+  include FilterByOwner
+  include FilterByCurrentUserRoles
+  # include TranslatableResource
+  include IndexSearch
+  include AdminAllowed
+
+  require_authentication :update, :create, :destroy, scopes: [:organization]
+
+  resource_actions :show, :index, :create, :update, :deactivate
+  schema_type :json_schema
+
+  prepend_before_action :require_login,
+    only: [:create, :update, :destroy]
+
+  def create
+    organizations = Organization.transaction do
+      Array.wrap(params[:organizations]).map do |organization_params|
+        Organizations::Create.with(api_user: api_user).run!(organization_params)
+      end
+    end
+
+    created_resource_response(organizations)
+  end
+
+  def update
+    Organization.transaction do
+      Array.wrap(resource_ids).zip(Array.wrap(params[:organizations])).map do |organization_id, organization_params|
```

Comment:
Yeah - it was a JSON-API spec requirement from memory. We do have [code](https://github.com/zooniverse/Panoptes/blob/922b5d0548a9362584d0ce592089089d75d8f6d7/lib/role_control/roled_controller.rb#L44) to handle the multi ids, however I'm still not sure they make it to the controller layer from the routes we've specified, e.g.
`PUT  |  /api/users/:id  |  api/v1/users#update  |  {:id=>/[0-9]*/}`

Agree we could look to take these out but not for this PR, so happy for this to go through as is and we'll revisit later.